example/integration_test/bucketeer_flutter_sdk_test.dart

```diff
@@ -322,33 +329,72 @@ void main() async {
-            (WidgetTester _) async {
-          final listener = MockEvaluationUpdateListener();
-          final listenToken =
+        (WidgetTester _) async {
+      final listener = MockEvaluationUpdateListener();
+      final listenToken =
           BKTClient.instance.addEvaluationUpdateListener(listener);
-          // Make sure `listener.onUpdate()` called
-          // Wait for all evaluations fetched by the SDK automatically after `initialize`
-          // We will be ready to run specific tests after the `listener.onUpdate()` is called.
-          // Use Completer to convert a listener callback to a future
-          var completer = Completer();
-          when(() => listener.onUpdate()).thenAnswer((invocation) {
-            //Called, complete the future
-            completer.complete();
-          });
-          await completer.future.timeout(const Duration(seconds: 60),
-              onTimeout: () {
-                // Fast fail
-                fail("The OnUpdate callback should called under 60 seconds");
-              });
-          var onUpdateCallCount = verify(() => listener.onUpdate()).callCount;
-          // The listener should called 1 times.
-          expect(onUpdateCallCount, 1,
-              reason:
+      // Make sure `listener.onUpdate()` called
+      // Wait for all evaluations fetched by the SDK automatically after `initialize`
+      // We will be ready to run specific tests after the `listener.onUpdate()` is called.
+      // Use Completer to convert a listener callback to a future
+      var completer = Completer();
+      when(() => listener.onUpdate()).thenAnswer((invocation) {
+        //Called, complete the future
+        completer.complete();
+      });
+      await completer.future.timeout(const Duration(seconds: 60),
+          onTimeout: () {
+        // Fast fail
+        fail("The OnUpdate callback should called under 60 seconds");
+      });
+      var onUpdateCallCount = verify(() => listener.onUpdate()).callCount;
+      // The listener should called 1 times.
+      expect(onUpdateCallCount, 1,
+          reason:
               "The OnUpdate callback should called when the evaluations are updated");
-          // Check remove the listener. If the `removeEvaluationUpdateListener` fail, the test will fail.
-          // The `completer` instance may get more call more times.
-          // Because it already complete, it will throw an exception cause the test fail.
-          BKTClient.instance.removeEvaluationUpdateListener(listenToken);
-        });
+      // Check remove the listener. If the `removeEvaluationUpdateListener` fail, the test will fail.
+      // The `completer` instance may get more call more times.
+      // Because it already complete, it will throw an exception cause the test fail.
+      BKTClient.instance.removeEvaluationUpdateListener(listenToken);
+    });
+  });
+
+  test('BKTClient should allow feature_tag to be optional',
```

Comment:
Did the e2e test pass?

Reply:
the Android e2e passed. I am looking on the iOS

Reply:
#### iOS e2e test passed

<img width="1044" alt="Screenshot 2023-10-17 at 15 03 34" src="https://github.com/duyhungtnn/bkt-flutter/assets/2597710/d1825d7a-1100-4e30-93b3-86ef6312574d">

#### Android e2e test passed


<img width="905" alt="Screenshot 2023-10-17 at 15 10 41" src="https://github.com/duyhungtnn/bkt-flutter/assets/2597710/84ca8ee6-bd7a-4dc2-8387-d939ea3b47b3">